database/models/event_rating.ts

```diff
@@ -4,21 +4,22 @@ import type { Events, EventsId } from './events';
 import type { Users, UsersId } from './users';
 
 export interface EventRatingAttributes {
-  id?: number;
```

Comment:
Turns out I'm getting those changes as well, so I've committed them now. 

Must be caused by the changes here (`sequelize-typescript` 1 -> 2 major version update):
https://github.com/mbrizic/ratethegame/pull/21/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R29

Reply:
That''s probably it then! I'm on version `2.1.1`